### PR TITLE
(dont unfollow active profile feature) xpath update, graphql information collect update

### PR DESCRIPTION
**Pull request by converge**
- xpath update
- graphql information collect update

### CODE REVIEW

File: instapy/util.py [hunk 1]
```diff
@@ -546,9 +540,8 @@ def get_active_users(browser, username, posts, boundary, logger):
             sleep_actual(2)
             try:
                 likers_count = browser.execute_script(
-                    "return window._sharedData.entry_data."
-                    "PostPage["
-                    "0].graphql.shortcode_media.edge_media_preview_like.count")
+                    "return window._sharedData.entry_data.ProfilePage[0]"
+                    ".graphql.user.edge_saved_media.count")
```
Comment (breuerfelix) [inline instapy/util.py:544]: I didn't know that this is possible... concat string without a `+` wow you never stop learning :D

I like the way you place whitespace !

**Review by breuerfelix — APPROVED** [review]

**Comment by breuerfelix** [comment]
would be cool to change base branch to dev :) to follow the way dev -> master

File: instapy/util.py (hunk 1 above)
Comment (converge) [reply instapy/util.py:544]: thanks :D
that's the spirit, always moving forward, always learning

**Comment by converge** [comment]
> would be cool to change base branch to dev :) to follow the way dev -> master

suure, I forgot about that. I will do it ater

**Comment by converge** [comment]
https://github.com/timgrossmann/InstaPy/pull/4678

**Pull request closed without merging**